pages.de/linux/see.md

```diff
@@ -0,0 +1,12 @@
+# see
```

Comment:
Just for your information, I just saw that this page needs a rewrite, which also includes your translation.

Reply:
oh well, good to know I guess :)